src/PatientChecking/PatientChecking/Views/Patient/Detail.cshtml

```diff
@@ -61,14 +78,35 @@
                 <a href="#">Contact detail</a>
             </div>
         </div>
+
+        <div id="myModal" class="modal">
+            <div class="modal-content-container">
+                <div class="modal-header">
+                    <p>Upload Patient Image</p>
+                    <span class="close">&times;</span>
+                </div>
+                <div class="upload-image-container">
+                    @using (Html.BeginForm("UploadImage", "Patient", FormMethod.Post,
+                new { enctype = "multipart/form-data", @class = "upload-form" }))
```

Comment:
Please move this line to above

Reply:
This line has been moved to above. Please review this @vietvo361